hypothesis-python/docs/stateful.rst

```diff
@@ -212,13 +237,15 @@ instead.
 
 .. autofunction:: hypothesis.stateful.multiple
 
+.. autoclass:: hypothesis.stateful.Bundle
+
 -----------
 Initializes
 -----------
 
-Initializes are a special case of rules that are guaranteed to be run at most
-once at the beginning of a run (i.e. before any normal rule is called).
-Note if multiple initialize rules are defined, they may be called in any order,
+Initializes are a special case of rules, which are guaranteed to be run exactly
+before any normal rule is called.
```

Comment:
There seems to be a word missing here, "exactly **once**"

Reply:
whoops, Anne added that and then I deleted it while reformatting 🤦 
Thanks for the catch!